client/src/js/base/InitialIcon.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import styled from "styled-components";
+import { getColor, getFontWeight, theme } from "../app/theme";
+
+export const StyledInitialIcon = styled.span`
+    border-radius: 50%;
+    display: flex;
+    flex: 0 0 auto;
+    justify-content: center;
+    align-items: center;
+    height: ${props => `${props.size}px`};
+    width: ${props => `${props.size}px`};
+    background: ${props => `hsl(${props.hash}, 83%, 21%);`}
+    color: ${getColor({ color: "white", theme })};
+    font-weight: ${getFontWeight("thick")};
+    font-size: ${props => `${props.size / 1.95}px`};
+`;
```

Comment:
Let's tighten this up.

1. Allow predefined sizes instead of taking `size` in `px` (eg. `sm`, `md`, `lg`). This should fit well in whatever `fontSize` theme value they match with. In other words, a small `InitialIcon` should fit well next to text with `sm` font size.
2. Focus on `font-size` for sizing these. Maybe just use the themed font sizes.
3. Don't use `flex`. Just set varying padding on the `span` depending on `span` to get to the final circle size.

Let me know if you need to discuss.

Reply:
Updated to use the same string sizes as used in theme rather than integers. As discussed, new version still uses flexbox due to consistent sizing problems of the padding approach. As suggested in point 2 the font sizes used are the same as those in the theme. Most of the time this looks good, but in the `Account/Profile` view a larger version seemed more fitting.